feature/ruledetail/src/androidTest/java/com.merxury.blocker.feature.ruledetail/RuleDetailScreenTest.kt

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2023 Blocker
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.merxury.blocker.feature.ruledetail
```

Comment:
The folder name for this file is wrong.

`com.merxury.blocker.feature.ruledetail/RuleDetailScreenTest.kt`

it should be 

`com/merxury/blocker/feature/ruledetail/RuleDetailScreenTest.kt`